src/main/resources/reference.conf

```diff
@@ -346,6 +346,9 @@ resourceTypes = {
       delete = {
         description = "delete this billing-project"
       }
+      update = {
```

Comment:
actions should be as granular as possible so it is easier to create roles that do just what we want. So this should be `update_billing_account`

Reply:
I was thinking that we may want an update billing project endpoint (that would cover full updates), and this could serve as the permission for it all, but that may be too much future proofing.